CONTRIBUTING.md

```diff
@@ -32,4 +32,4 @@ GitHub Pages runs off the `gh-pages` branch, which can only be pushed to by the
 
 ## Release
 
-The maintainers will release new versions to NPM as appropriate. The procedure that the maintainers will use to release is to update the changelog with the newest version, update the package.json version, and tag the main branch with the version number, upon which the travis-CI will run tests again before deploying to NPM automatically.
\ No newline at end of file
+The maintainers will release new versions to NPM as appropriate. The procedure that the maintainers will use to release is to update the changelog with the newest version, update the package.json version, and tag the main branch with the version number, upon which the travis-CI will run tests again before deploying to NPM automatically.
```

Comment:
This line shows up in the diff, but it doesn't seem like you made any changes -- something going on with a newline, maybe?

Reply:
Yeah, vim adds a new line to the end of the file. (I think it's a feature I can actually shut off)